src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,107 @@
+package Duke;
+
+import Duke.Exceptions.DukeException;
+import Duke.Tasks.TaskList;
+import Duke.Tasks.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.lang.String;
+
+/**
+ * Contains Ui object that deals with interactions with the user
+ */
+
+public class Ui {
+    private final static String UNDERLINE = "________________________________________________________________";
+    private final static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private Scanner text = new Scanner(System.in);
+
+    /**
+     * The method of Underline() to print out a separating line
+     */
+    public static void Underline(){
```

Comment:
To follow coding standard.
```suggestion
    public static void underline()
```